sdk/generic-sdk-core/generic-core/src/main/java/com/generic/core/implementation/http/RestProxy.java

```diff
@@ -3,57 +3,58 @@
 
 package com.generic.core.implementation.http;
 
+import com.generic.core.http.models.HttpRequest;
+import com.generic.core.http.models.RequestOptions;
 import com.generic.core.http.pipeline.HttpPipeline;
 import com.generic.core.implementation.http.rest.RestProxyImpl;
 import com.generic.core.implementation.http.rest.RestProxyUtils;
 import com.generic.core.implementation.http.rest.SwaggerInterfaceParser;
 import com.generic.core.implementation.http.rest.SwaggerMethodParser;
-import com.generic.core.http.models.RequestOptions;
 import com.generic.core.util.serializer.ObjectSerializer;
 
 import java.lang.reflect.InvocationHandler;
 import java.lang.reflect.Method;
 import java.lang.reflect.Proxy;
 
 /**
- * Type to create a proxy implementation for an interface describing REST API methods.
- * <p>
- * RestProxy can create proxy implementations for interfaces with methods that return deserialized Java objects as well
- * as asynchronous Single objects that resolve to a deserialized Java object.
+ * Type to create a {@link Proxy} implementation for an interface describing REST API methods.
+ *
+ * <p>{@link RestProxy} can create {@link Proxy} implementations for interfaces with methods that return deserialized
+ * Java objects as  * well as asynchronous single objects that resolve to a deserialized Java object.</p>
  */
 public final class RestProxy implements InvocationHandler {
     private final SwaggerInterfaceParser interfaceParser;
-    private final HttpPipeline httpPipeline;
     private final RestProxyImpl restProxyImpl;
 
     /**
-     * Create a RestProxy.
+     * Create a {@link RestProxy}.
```

Comment:
Focus on the impl - there is no point tweaking a bunch of JavaDoc for implementation code that won't ever have its javadoc generated. The end result is just a bunch of PR noise.